packages/core/test/SchemaField_test.js

```diff
@@ -20,6 +20,66 @@ describe("SchemaField", () => {
     sandbox.restore();
   });
 
+  describe("registry", () => {
+    it("should provide expected registry as prop", () => {
+      let receivedProps;
+      const schema = {
+        type: "object",
+        definitions: {
+          a: { type: "string" },
+        },
+      };
+      createFormComponent({
+        schema,
+        uiSchema: {
+          "ui:field": props => {
+            receivedProps = props;
+            return null;
+          },
+        },
+      });
+
+      const { registry } = receivedProps;
+      expect(registry).eql({
+        widgets: getDefaultRegistry().widgets,
+        fields: getDefaultRegistry().fields,
+        definitions: schema.definitions,
+        rootSchema: schema,
+        ArrayFieldTemplate: undefined,
+        FieldTemplate: undefined,
+        ObjectFieldTemplate: undefined,
+        formContext: {},
+      });
+    });
+    it("should set definitions to empty object if it is undefined", () => {
+      let receivedProps;
+      const schema = {
+        type: "object",
+      };
+      createFormComponent({
+        schema,
+        uiSchema: {
+          "ui:field": props => {
+            receivedProps = props;
+            return null;
+          },
+        },
+      });
+
+      const { registry } = receivedProps;
+      expect(registry).eql({
+        widgets: getDefaultRegistry().widgets,
+        fields: getDefaultRegistry().fields,
+        definitions: {},
+        rootSchema: schema,
+        ArrayFieldTemplate: undefined,
+        FieldTemplate: undefined,
+        ObjectFieldTemplate: undefined,
+        formContext: {},
+      });
+    });
+  });
+
```

Comment:
Small comment, feel free to close it yourself.

I would focus on just the tested code, eg instead of : 

```
      const { registry } = receivedProps;
      expect(registry).eql({
        widgets: getDefaultRegistry().widgets,
        fields: getDefaultRegistry().fields,
        definitions: {},
        rootSchema: schema,
        ArrayFieldTemplate: undefined,
        FieldTemplate: undefined,
        ObjectFieldTemplate: undefined,
        formContext: {},
      });
```

I would do : `expect(receivedProps.registry.definitions).eql({});`

Reply:
Good point -- in this case, I added this because we had no code at all that tested that all the right keys are present in the registry (because this PR, for example, adds the key `rootSchema` to the registry.)